Makefile

```diff
@@ -315,7 +350,8 @@ gfx/mobile/stadium2_n64.2bpp: tools/gfx += --trim-whitespace
 		tools/gfx $(tools/gfx) --depth 1 -o $@ $@)
 
 %.gbcpal: %.png
-	$(RGBGFX) --colors embedded -p $@ $<
+	$(RGBGFX) -p $@ $<
+	tools/gbcpal $(tools/gbcpal) $@
```

Comment:
Make this `tools/gbcpal $(tools/gbcpal) $@ || { rm -f $@; false }`, to remove the file and stop on error (or make a separate rule...).

Reply:
Can you please explain further what this would be protecting against? Note that if `tools/gbcpal` has an error, it would exit before the opening the file for writing with `write_u8` at the very end.

Reply:
The file won't be deleted if tools/gbcpal fails. Running "make" again will consider the file to be up to date, and cause the partially-generated file to be used to build the rest.
It's a common makefile footgun.

Reply:
Makes sense, thank you! I updated the `tools/gfx` rules likewise.